UPGRADE-1.4.md

```diff
@@ -1,9 +1,44 @@
 # UPGRADE FROM `v1.3.X` TO `v1.4.0`
 
-* Deprecated not passing `Sylius\Component\Locale\Context\LocaleContextInterface` instance as the second argument 
-  to `Sylius\Bundle\LocaleBundle\Templating\Helper\LocaleHelper`'s constructor.
 
-* Due to switch to `doctrine/doctrine-migrations-bundle:^2.0`, some changes in migration files are required:
-    * `Doctrine\DBAL\Migrations\AbstractMigration` -> `Doctrine\Migrations\AbstractMigration`
-    * `Doctrine\DBAL\Migrations\SkipMigrationException` -> `Doctrine\Migrations\Exception\SkipMigration` (if used)
-    * `: void` return type added to both `up` and `down` methods
+First step is upgrading Sylius with composer
+
+- `composer require sylius/sylius:~1.4.0 --no-update`
+
+### Doctrine migrations
+
+* Change base `AbstractMigration` namespace to `Doctrine\Migrations\AbstractMigration`
+* Add `: void` return types to both `up` and `down` functions
+* Copy [this](https://github.com/Sylius/Sylius-Standard/blob/1.4/src/Migrations/Version20190109095211.php) and [this](https://github.com/Sylius/Sylius-Standard/blob/1.4/src/Migrations/Version20190109160409.php) migration to your migrations folder or run `doctrine:migrations:diff` to generate new migration with changes from **Sylius**
+
+### Dotenv
+
+* `composer require symfony/dotenv:^4.2 --dev --no-update`
+* Follow [Symfony dotenv update guide](https://symfony.com/doc/current/configuration/dot-env-changes.html) to incorporate required changes in `.env` files structure. Optionally, you can take a look at [corresponding PR](https://github.com/Sylius/Sylius-Standard/pull/323) introducing these changes in **Sylius-Standard**
+* If you're using our Travis CI configuration, follow [this changes](https://github.com/Sylius/Sylius-Standard/pull/323/files#diff-354f30a63fb0907d4ad57269548329e3) introduced in `.travis.yml` file
+
+At the end, update libraries with `composer update` and run apply migrations with `bin/console doctrine:migrations:migrate`.
+
+Don't forget to clear the cache (`bin/console cache:clear`) to be 100% everything is loaded properly.
+
+---
+
+### Behat
+
+If you're using Behat and want to be up-to-date with our configuration
```

Comment:
We can mention FOB's guide for migration also - https://github.com/FriendsOfBehat/SymfonyExtension/blob/master/UPGRADE-2.0.md